### DISCUSSION TITLE
container cannot create pgbackup

### DISCUSSION THREAD

**Discussion by davidsugianto** [prodrigestivill/docker-postgres-backup-local#111] [Q&A] [answered]
pgbackup             | 2019/08/08 13:42:35 Opening port 80 for health checking
pgbackup             | 2019/08/08 13:45:35 11 cmd: /backup.sh 
pgbackup             | 2019/08/08 13:45:35 11: Missing POSTGRES_DB_FILE file.
pgbackup             | 2019/08/08 13:45:35 11 Exit Status: 1

**Comment by prodrigestivill** [comment, marked as answer]
You need to setup the environment variables acordingly as described in the README.md.

If you want to use POSTGRES_DB_FILE, you need to also setup the volume with the referenced file.

For more information lookup for official docker and postgres docker container documentation.
